python/datasources/acs_health_insurance.py

```diff
@@ -0,0 +1,657 @@
+    "_048E": meta(Sex.FEMALE, 45, 54, HealthInsurancePopulation.WITHOUT),
+    "_049E": meta(Sex.FEMALE, 55, 64),
+    "_050E": meta(Sex.FEMALE, 55, 64, HealthInsurancePopulation.WITH),
+    "_051E": meta(Sex.FEMALE, 55, 64, HealthInsurancePopulation.WITHOUT),
+    "_052E": meta(Sex.FEMALE, 65, 74),
+    "_053E": meta(Sex.FEMALE, 65, 74, HealthInsurancePopulation.WITH),
+    "_054E": meta(Sex.FEMALE, 65, 74, HealthInsurancePopulation.WITHOUT),
+    "_055E": meta(Sex.FEMALE, 65, None),
+    "_056E": meta(Sex.FEMALE, 65, None, HealthInsurancePopulation.WITH),
+    "_057E": meta(Sex.FEMALE, 65, None, HealthInsurancePopulation.WITHOUT),
+}
+
+# Given the Prefix: eg.
+# {"C27001A":{MetadataKey.RACE: Race.WHITE_ALONE}}
+# will combine with the provided suffixes eg.
+# {
+#   "_002E":meta(None,0, 19),
+#   "_003E":meta(None,0, 19,HealthInsurancePopulation.WITH)
+#   ...
+# }
+# to create the ACS variable params.
+# ?for=state&get=C27001A_002E,C27001A_003E...
+
+
+def format_params(prefixes, suffixes, isCounty=False):
+    groups = []
+    for prefix in prefixes:
+        for suffix in suffixes:
+            groups.append(prefix+suffix)
+    vars = ','.join(groups)
+
+    return {'for': 'county' if isCounty else 'state', "get": vars}
+
+# Helper to log response codes from ACS calls
+
+
+def log_response(resp):
+    print("-"*50)
+    print(f'Get Request ({resp.status_code}): {resp.url}')
+    print("-"*50)
+
+
+class AcsHealhInsuranceIngestor:
+
+    # Initialize variables in class instance, also merge all metadata so that lookup of the
+    # prefix, suffix combos can return the entire metadata
+    def __init__(self, base):
+        self.baseUrl = base
+        self.data = {}
+        self.metadata = {}
+        self.buildMetadataList()
+
+    # Gets standardized filename
+    def getFilename(self, sex, race, isCounty):
+        if(race is not None):
+            return "HEALTH_INSURANCE_BY_RACE_{0}_{1}.json".format("STATE" if isCounty else "COUNTY", race)
+        else:
+            return "HEALTH_INSURANCE_BY_SEX_{0}_{1}.json".format("STATE" if isCounty else "COUNTY", sex)
+
+    # Method to output <Filename.csv>.  Used for debugging purposes.
+    def write_local_files_debug(self):
+        self.getStateFipsMap()
+        self.getCountyFipsMap()
+        self.getHealthInsuranceDataBySex()
+        self.getHealthInsuranceDataByRace()
+        self.splitDataFrames()
+
+        with open('total_health_insurance.json', 'w') as f:
+            print(json.dumps(
+                {str(k): v for k, v in self.data.items()}, indent=4), file=f)
+
+        for table_name, df in self.frames.items():
+            df.to_csv(table_name + ".csv", index=False)
+
+    # Uploads the ACS data to GCS by providing
+    # the ACS Base URL
+    # Acs Query Params
+    # Standardized Filename
+    #
+    # An example file created in GCS:
+    # HEALTH_INSURANCE_BY_RACE_COUNTY_WHITE_ALONE.json
+
+    def upload_to_gcs(self, bucket):
+        maleStateParams = format_params(
```

Comment:
I believe the python style guide (at least within google) says to use underscored names for all local variables rather than camel case, e.g. male_state_params instead of maleStateParams. Would you mind updating all of your local variables to follow this style?